src/app/providers/providers.module.ts

```diff
@@ -35,6 +36,7 @@ import { ConfiguredAzureDevOpsModule, ConfiguredGitHubApiModule, ConfiguredGitLa
     ConfiguredAzureDevOpsModule,
     ConfiguredGitHubApiModule,
     ConfiguredGitLabApiModule,
+    TagModule,
```

Comment:
The `Configure*Module` are a little out of place as well, but IMO this list should be sorted.

*(I cannot add a suggestion code block for unaltered code lines.)*

Reply:
Now sorted alphabetically.